public/style/sass/_posts.scss

```diff
@@ -32,13 +34,29 @@
   background-color: white;
 }
 
+// BOXES
+
+.view {
+  @include flexbox;
+  color: $waitrose-green;
+  font: bold 17px $default;
+}
+
+.map-icon {
+  margin-right: 10px;
+}
+
+.img {
+  padding: 15px;
+}
+
 .size {
   text-align: center;
   font: bold 17px $default;
   color: $waitrose-green;
 }
 
-.post-box.tags {
+.tags {
```

Comment:
I wonder if the tags would look better smaller. Feels messy them spread across two lines. I think they could get away with being smaller and poss with a bit of border radius too. 

Feel free to disregard this though, just a personal stylistic comment, not worth spending too long on if you disagree